libraries/chain/asset_evaluator.cpp

```diff
@@ -56,34 +58,38 @@ void_result asset_create_evaluator::do_evaluate( const asset_create_operation& o
    if( d.head_block_time() > HARDFORK_385_TIME )
    {
 
-   if( d.head_block_time() <= HARDFORK_409_TIME )
-   {
-      auto dotpos = op.symbol.find( '.' );
-      if( dotpos != std::string::npos )
+      if( d.head_block_time() <= HARDFORK_409_TIME )
       {
```

Comment:
Logically, the code in this `if` block, even this `if` check, can be removed. Submitted issue https://github.com/bitshares/bitshares-core/issues/712.